README.md

```diff
@@ -16,6 +16,10 @@ ______________________________________________________________________
 
 </div>
 
+## 🔍 Preview
+
+Feel free to checkout our Streamlit application at at https://head-segmentation-d4mc3a3f3q-lm.a.run.app/
```

Comment:
```suggestion
Feel free to checkout our Streamlit application at https://head-segmentation-d4mc3a3f3q-lm.a.run.app/
```

Reply:
Btw, are you 300% sure that it won't create any unnecessary costs on your side?

Reply:
Yeah! For the next 3 months we have 1200 zł on GCP. So we can have it running for a while and see how much of a cost does it generate. We'll decide on 'sponsoring' it after the trial period